charts/cloudzero-agent/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0]
+- Remove Node Exporter dependencies.
+- Dynamically populate service endpoints for Node Exporter.
```

Comment:
should we replace `Node Exporter` with `Kube State Metrics`?

Reply:
Yes, thanks for catching this.